.github/actions/spelling/line_forbidden.patterns

```diff
@@ -270,12 +476,12 @@
 \b(?:coarse|fine) grained\b
 
 # Homoglyph (Cyrillic) should be `A`/`B`/`C`/`E`/`H`/`I`/`I`/`J`/`K`/`M`/`O`/`P`/`S`/`T`/`Y`
-# It's possible that your content is intentionally mixing Cyrllic and Latin scripts, but if it isn't, you definitely want to correct this.
```

Comment:
I've added testing to catch when I make typos ☺️  in this file so there should be fewer of them going forward...